.github/ISSUE_TEMPLATE/update-base-java.md

```diff
@@ -12,8 +12,17 @@ assignees: ''
 
 Part of #xxx.
 
+<!--
+This gives hints to the person doing the work.
+Add/Change/Remove anything that isn't applicable anymore
+-->
+- Add: `x.x.x`
+- Remove: `y.y.y`
+
 > [!TIP]
-> Please add the `scheduled-for/20XX-XX` label.
+> Please add the `scheduled-for/20XX-XX` label, and add to the [Stackable Engineering][1] project.
```

Comment:
note: Maybe we should leave a comment here explaining why the ticket needs manual addition to the board even tho the project is mentioned in the frontmatter.

Reply:
Done in 7548c7a6e6c2086834612aeeaa99f25bc831b9e7